R/utils.R

```diff
@@ -182,3 +182,116 @@ is_installed <- function(path_pkg = ".") {
 is_windows <- function() {
   identical(.Platform$OS.type, "windows")
 }
+
+#' Pin asset directory
+#'
+#' @description Add directories that need to be available on both branches
```

Comment:
You can remove the `@description` roxygen tag, it is recognized automatically, just as you don't need `@title` for the title. Can you change this everywhere?